robot/Cumulus/resources/NPSP.robot

```diff
@@ -356,18 +356,21 @@ Validate And Create Required CustomField
     ...                    Run keyword to create custom field based on the field type selection
     [Arguments]            &{fields}
     Load Page Object                                     Custom                           ObjectManager
-    Open Fields And Relationships                        &{fields}[Object]
-    Run Keyword If     '&{fields}[Field_Type]' == "Lookup"   Create Custom Field
-    ...                                                      &{fields}[Field_Type]
-    ...                                                      &{fields}[Field_Name]
-    ...                                                      &{fields}[Related_To]
-    Run Keyword If     '&{fields}[Field_Type]' == "Currency"   Create Custom Field
-    ...                                                      &{fields}[Field_Type]
-    ...                                                      &{fields}[Field_Name]
-    Run Keyword If     '&{fields}[Field_Type]' == "Formula"   Create Custom Field
-    ...                                                      &{fields}[Field_Type]
-    ...                                                      &{fields}[Field_Name]
+    Open Fields And Relationships                        ${fields}[Object]
+    Run Keyword If     '${fields}[Field_Type]' == "Lookup"   Create Custom Field
+    ...                                                      ${fields}[Field_Type]
+    ...                                                      ${fields}[Field_Name]
+    ...                                                      ${fields}[Related_To]
+    Run Keyword If     '${fields}[Field_Type]' == "Currency"  Create Custom Field
+    ...                                                      ${fields}[Field_Type]
+    ...                                                      ${fields}[Field_Name]
+    Run Keyword If     '${fields}[Field_Type]' == "Formula"  Create Custom Field
+    ...                                                      ${fields}[Field_Type]
+    ...                                                      ${fields}[Field_Name]
     ...                                                      formula=&{fields}[Formula]
+    Run Keyword If     '${fields}[Field_Type]' == "Text"     Create Custom Field
+    ...                                                      ${fields}[Field_Type]
+    ...                                                      ${fields}[Field_Name]
```

Comment:
In the object manager minor changes PR I've got that I made as per Bryan's suggestion, I removed al the Run keyword if statements and directly called the create custom Field. So we might have to catch up here

Reply:
yes, u still need to merge the PR right? once it is merged I will update my branch with latest code and will update accordingly